src/main/resources/db/migration/h2/V1.2__oauth_code.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE `oauth_code` (
+  `code` VARCHAR(255) PRIMARY KEY,
+  `authentication` LONGVARBINARY
```

Comment:
it seems like `blob` is right datatype for H2
http://www.h2database.com/html/datatypes.html#blob_type